flask_static_digest/digester.py

```diff
@@ -6,12 +7,16 @@
 import re
 import shutil
 
+import brotli
+
 DIGESTED_FILE_REGEX = r"-[a-f\d]{32}"
+CHUNK_SIZE = 1024 * 64
```

Comment:
Just for my own knowledge, what this size control exactly?

Reply:
I missed this earlier - brotli lacks the file based API call that gzip has which you used. To avoid pulling a whole file into memory and writing it out in one shot (as the file could be very large) this chunks it through a streaming compressor. I might want to increase that chunk size a lot though, 64k is a bit small on any Python host! There is a finish command in there to spit out any last bits less than on chunk in size.